### DISCUSSION TITLE
My friend has not received an answer from GitHub Support: Ticket ID 2693772

### DISCUSSION THREAD

**Discussion by Vivienne-00** [community/community#120992] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My friend (claudiu-sava) got disabled on 25. March without receiving an email notice or an explanation. The disable reasons are unclear. He created a support ticket asking about the reasons and he has not received an answer to his Ticket ID: 2693772. Can you please take action and investigate this problem?
Thank you in advance and best Regards.

**Comment by ThomasLaforge** [comment]
Same for my students... So bad
https://github.com/orgs/community/discussions/116740

**Comment by mathieujelsch** [comment]
Same for me, I am waiting for an answer since 2 weeks, I got this message : You are marked as spam, and therefore cannot authorize a third party application. What happen to the Github support ?

**Comment by mathieuj2024** [comment]
Hello it's the same for me, I am waiting since 2 weeks, I got any answer. I had to create a new account just to write this message. I got this message on my account : You are marked as spam, and therefore cannot authorize a third party application. My Ticket number is #2723475. What happen to the Github support ?

**Comment by ebndev** [comment]
Hi @Vivienne-00,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
